.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.4.0
+current_version = 3.6.0
```

Comment:
3.4 to 3.6?

Reply:
Unfortunately yes, this file didn't update previously. Current version is 3.5 so this brings it back into line https://pypi.org/project/opentok/